src/components/lib/VTextField/VTextField.vue

```diff
@@ -118,6 +118,7 @@ function onIconClicked(ev) {
         :class="{ strong }"
         :style="inputStyle"
         @input="onInput"
+        @keyup.enter.prevent=""
```

Comment:
Why do we need this?

Reply:
Removed this. Actually this VTextField component is a `div` consisting of a `label` element and `input` element. I've also added `v-bind="attrs"` to input so any additional attributes like `placeholder` gets added to input element. 

Now when I add `@keyup.enter` on VTextField, it gets added to input element as well as the parent div that holds this input element. So this event is triggered twice in this case. To tackle that problem, Vue 2 has an option called `inheritAttrs: false` which disallows parent div to inherit additional attributes other than props, but unfortunately in Vue 3 we dont have that option. There are workarounds for that but to do that we'll need to disable linter for this file

Reply:
You can set `inheritAttrs: false` using a separate script tag.
See: https://vuejs.org/guide/components/attrs.html#disabling-attribute-inheritance

```
<script>
export default { inheritAttrs: false }
</script>

<script setup>
... component setup ...
</script>
```